test/integration/console_access.go

```diff
@@ -0,0 +1,64 @@
+// +build integration,!no-etcd
+
+package integration
+
+import (
+	"crypto/tls"
+	"net/http"
+	"testing"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	testutil "github.com/openshift/origin/test/util"
+)
+
+func TestAccessOriginWebConsole(t *testing.T) {
```

Comment:
ok... I'm confused... should this test file go away? looks like you have the same thing below